packages/frontend-shared/src/locales/en-US.json

```diff
@@ -327,7 +327,27 @@
     },
     "experiments": {
       "title": "Experiments",
-      "description": "If you'd like to try out new features that we're working on, you can enable beta features for your project by turning on the experimental features you'd like to try. {0}"
+      "description": "If you'd like to try out new features that we're working on, you can enable beta features for your project by turning on the experimental features you'd like to try. {0}",
+      "experimentalFetchPolyfill": {
+        "name": "Fetch Polyfill",
+        "description": "Automatically replaces window.fetch with a polyfill that Cypress can spy on and stub"
+      },
+      "experimentalInteractiveRunEvents": {
+        "name": "Interactive Run Events",
+        "description": "Changes the way some events can be listened to in plugins"
+      },
+      "experimentalSessionSupport": {
+        "name": "Session Support",
+        "description": "Enable use of cy.session()"
+      },
+      "experimentalSourceRewriting": {
+        "name": "AST based test Source Rewriting",
+        "description": "This may fix issues caused by the existing regex-based JS/HTML replacement algorithm"
+      },
+      "experimentalStudio": {
+        "name": "Studio Mode",
+        "description": "Use cypress GUI to write tests."
+      }
```

Comment:
Can someone from DX vet those descriptions?

Reply:
I can do it.